website/content/docs/api-clients/api/index.mdx

```diff
@@ -82,196 +82,13 @@ The following method conventions are used within Boundary's API:
-
-```hcl
-controller {
-  api_rate_limit {
-    resources = ["target"]
-    actions   = ["list"]
-    per       = "auth-token"
-    limit     = 10
-    period    = "1s"
-  }
-}
-```
-
-The following example is more complex.
-Initially it sets some defaults to apply to all resources and actions.
-Then it configures some specific endpoints with different limits.
-
-```hcl
-controller {
-  # Total limit for all resources and actions
-  api_rate_limit {
-    resources = ["*"]
-    actions   = ["*"]
-    per       = "total"
-    limit     = 500
-    period    = "1s"
-  }
-
-  # Total limit for all list operations,
-  # sets an overall cap on the list action since it is relatively expensive
-  api_rate_limit {
-    resources = ["*"]
-    actions   = ["list"]
-    per       = "total"
-    limit     = 200
-    period    = "1s"
-  }
-
-  # Limit for IP address to all resources and actions
-  # to prevent a malicious host that is fabricating tokens
-  # or spamming unauthed endpoints
-  api_rate_limit {
-    resources = ["*"]
-    actions   = ["*"]
-    per       = "ip-address"
-    limit     = 50
-    period    = "1s"
-  }
-
-  # Limit of all authed requests, this essentially sets a default
-  # for all authed requests to prevent one user from consuming
-  # all of the total list limit set above. Any limits that follow
-  # can override the default for specific sets of resources and actions
-  api_rate_limit {
-    resources = ["*"]
-    actions   = ["*"]
-    per       = "auth-token"
-    limit     = 100
-    period    = "1s"
-  }
-
-  # Limit of all list operations by auth token to set a sane default
-  # since they are generally more expensive
-  api_rate_limit {
-    resources = ["*"]
-    actions   = ["list"]
-    per       = "auth-token"
-    limit     = 50
-    period    = "1s"
-  }
-
-  # Limit for targets and sessions list by auth token. These are resources
-  # that clients will want to list frequently, but might be more expensive
-  # if there is a large number of them for each user. Having a lower limit
-  # encourages the use of refresh tokens and caching.
-  api_rate_limit {
-    resources = ["target", "session"]
-    actions   = ["list"]
-    per       = "auth-token"
-    limit     = 20
-    period    = "1s"
-  }
-
-  #Limit for authorize session by auth token
-  api_rate_limit {
-    resources = ["target"]
-    actions   = ["authorize-session"]
-    per       = "auth-token"
-    limit     = 150
-    period    = "1s"
-  }
-}
-```
\ No newline at end of file
+When you search for Boundary resources using the `list` operation, you could potentially receive a very large number of results.
+Boundary uses an API pagination feature to enable you to search and filter large lists of search results without overwhelming your system resources.
+The results of the operation are cached locally, to help reduce server cost and further protect your system resources.
```

Comment:
Again, not sure this will cause confusion. The automatic caching is a feature of the CLI. The Desktop client also uses it, but it's not an automatic feature of the API itself. Maybe this should just be

```suggestion
When you search for Boundary resources using the `list` operation, you could potentially receive a very large number of results.
Boundary uses an API pagination feature to enable you to search and filter large lists of search results without overwhelming your system resources.
```

Reply:
> Is there already a different area where search and the caching is discussed?

Hi @talanknight , I started the work for search and caching on another branch, although I'm wondering if I should combine them now since it's all related. The draft PR is #4244 (it still needs work).